src/myMethods.js

```diff
@@ -0,0 +1,58 @@
+const prepareData = arr => {
+  return arr.map(person => {
+    const sortedPerson = {};
+
+    sortedPerson.id = arr.indexOf(person) + 1;
```

Comment:
```suggestion
    sortedPerson.id = i + 1;
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map